main/src/ca/uwaterloo/flix/language/errors/SafetyError.scala

```diff
@@ -164,6 +164,24 @@ object SafetyError {
     override def explain(formatter: Formatter): Option[String] = None
   }
 
+  /**
+   * An error raised to indicate that the Java class in a catch clause is not a Throwable.
+   *
+   * @param loc the location of the catch parameter.
+   */
+  case class IllegalCatchType(loc: SourceLocation) extends SafetyError with Recoverable {
+    def summary: String = s"Catch type is not a subclass of Throwable."
```

Comment:
Maybe `"Exception type is not a subclass of Throwable."` is more expect language?

Reply:
Agreed, and updated.